examples/program_generator/src/lib.rs

```diff
@@ -0,0 +1,29 @@
+#![no_std]
+
+extern crate alloc;
+
+use alloc::vec::Vec;
+use gstd::{prog::ProgramGenerator, CodeHash};
+
+fn salt_uniqueness_test() {
+    let n = 10;
+    let salts: Vec<_> = (0..n).map(|_| ProgramGenerator::get_salt()).collect();
+
+    for i in 0..n {
+        for j in (i + 1)..n {
+            assert_ne!(salts[i], salts[j]);
+        }
+    }
```

Comment:
As I said in previous impl, that's still irrational check!
Please, get vecs len, convert to set, get sets len and compare them!

Reply:
Fixed.